src/fread.c

```diff
@@ -2609,7 +2623,7 @@ int freadMain(freadMainArgs _args)
           if (tch != eof) tch++;
           myNrow++;
         }
-        if (verbose) { double now = wallclock(); thRead += now - tLast; tLast = now; }
+        if (verbose) { double now = wallclock(); timestamps.th_read += now - tLast; tLast = now; }
```

Comment:
And even if it did compile, this would create a mess as `timestamps.th_read` will then be shared by all threads. 

Inside the parallel loop, multiple threads would be trying to execute this line, reading and writing to the *same* shared variable at the *same* time. The reduction was supposed to prevent this race condition, but with this change it's being applied to the wrong variables.